models/users.go

```diff
@@ -0,0 +1,21 @@
+package models
+
+import "github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+
+type UserParams struct {
+	Forename string `json:"forename"`
+	Surname  string `json:"surname"`
+	Email    string `json:"email"`
+}
+type CreateUserInput struct {
+	UserInput *cognitoidentityprovider.AdminCreateUserInput
+}
+type CreateUserOutput struct {
+	UserOutput *cognitoidentityprovider.AdminCreateUserOutput
+}
+type ListUsersInput struct {
+	ListUsersInput *cognitoidentityprovider.ListUsersInput
+}
+type ListUsersOutput struct {
+	ListUsersOutput *cognitoidentityprovider.ListUsersOutput
+}
```

Comment:
Please can I ask is this a pattern that we should adopt for all our endpoints?

Reply:
I think so but feel free to disagree! A standardised way to model-ise our structs.